torch/optim/adam.py

```diff
@@ -76,7 +76,7 @@ def __setstate__(self, state):
                 if len(p_state) != 0 and not torch.is_tensor(p_state['step']):
                     step_val = float(p_state["step"])
                     p_state["step"] = (torch.tensor(step_val, dtype=_get_scalar_dtype(is_fused=fused), device=p.device)
-                                       if group['capturable'] or group['fused']
+                                       if group['capturable']
```

Comment:
may I know why you remove this `group['fused']` here?

Reply:
It's my mistake. I failed on some cuda UT and I mis-understand it. It should not be removed.